src/vos/vos_container.c

```diff
@@ -615,6 +646,31 @@ vos_cont_tab_register()
 	return rc;
 }
 
+int
+vos_dtx_update_resync_gen(daos_handle_t coh)
+{
+	struct vos_container	*cont;
+	struct vos_cont_df	*cont_df;
+	int			 rc;
+
+	cont = vos_hdl2cont(coh);
+	D_ASSERT(cont != NULL);
+
+	cont_df = cont->vc_cont_df;
+	rc = vos_tx_begin(vos_cont2umm(cont));
```

Comment:
No big deal, but this can just use pmemobj_memcpy_persist */

Reply:
OK